src/adapter/10/renderer.ts

```diff
@@ -699,7 +699,8 @@ export function createRenderer(
 				const c = getComponent(vnode);
 				if (c) {
 					const s = getHookState(c, index);
-					s[0] = value;
+					// TODO: How do we know it's okay to assign like this? Hook state varies.
```

Comment:
It's not clear from the types. This is based on the implicit knowledge that we only mark `useState` and `useReducer` as editable.

https://github.com/preactjs/preact-devtools/blob/c413b8a24e0142f7e776265f7eef782b328b4af0/src/adapter/10/renderer/hooks.ts#L173-L176